safety-quiz/covid.py

```diff
@@ -0,0 +1,26 @@
+# imports
+from flask import Flask, request, session, redirect, url_for, render_template, flash, send_from_directory, Markup, jsonify
+import sqlalchemy as sa
+from checkIn.model import User, UserLocation, Type, Training, Machine, Quiz, Question, Option, MissedQuestion, init_db, Major, College, HawkCard
+
+# Don't create new instance of app but use current one
+app = Flask(__name__)
+app.config.from_pyfile('config.cfg')
+db_session = init_db(app.config['DB'])
+app.config.update(dict(MAIL_SERVER = '10.0.8.18'),MAIL_DEFAULT_SENDER = "ideashop@iit.edu")
```

Comment:
better to break this out into config file

Reply:
Can you elaborate on what this means? I took these from nightly.py because I noticed that safety-quiz.py contains app setup, so I thought that nightly.py is using that same app so we don't need to generate several instances. How could I break this into config file?

Reply:
Best practice is to break out setup specific variables into a config file like the DB url is done in line 9 not like line 10. If I hard coded them into nightly.py we should also break those out into a config file for that file.

Reply:
Would this require a separate config for each file? Also, are these config files different than the ones currently existing? (config.cfg or config.cfg.default)

Reply:
you don't need a separate config file, config.cfg is where is should live, but config.cfg.default should add a line for any required config variables.

also, as written I'm not sure that this route is actually accessible to the flask app. will test shortly. I would recommend you check out this page: https://flask.palletsprojects.com/en/2.0.x/blueprints/
and how we used Blueprints in check-in-app here:
https://github.com/iit-ideashop/check-in-app/blob/master/checkIn/checkIn.py
https://github.com/iit-ideashop/check-in-app/blob/master/checkIn/controllers/auth.py